appveyor.yml

```diff
@@ -17,7 +17,9 @@ install:
   - npm install -g npm@5.5.1 yarn@1.3.2
   - pip install virtualenv==15.1.0
   - cinst sbt --version 1.0.2 -y
+  - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
   - refreshenv
+  - set PATH=C:\msys64\usr\bin;%PATH%
```

Comment:
Hmm, putting the whole of `C:\msys64\usr\bin` into `PATH` might lead to clashes, like with the `find` tool. However, I don't have a good idea how to address this right now.

Reply:
Would it be better to change the order to `%PATH%;C:\msys64\usr\bin`?

Reply:
Yeah, I guess that's a good compromise.